deploy-manage/upgrade/deployment-or-cluster/upgrade-on-ece.md

```diff
@@ -3,3 +3,46 @@ navigation_title: "Upgrade on {{ece}}"
+
+Once you're prepared to upgrade, do the following: 
+
+% Note: Add a link once confirmed where prepare to upgrade will reside in TOC. 
+
+1. Ensure your current ECE and Docker or Podman versions are [compatible](https://www.elastic.co/support/matrix/#elastic-cloud-enterprise) with the {{stack}} version you're upgrading to. For example, if you're upgrading to 9.0.0, the minimum required version is ECE 3.0. If you don’t have a compatible version installed, [upgrade your orchestrator](/deploy-manage/upgrade/orchestrator/upgrade-cloud-enterprise.md).  
+2. Download the most recent [stack pack](/deploy-manage/deploy/cloud-enterprise/manage-elastic-stack-versions.md#ece_most_recent_elastic_stack_packs) for the version you’re upgrading to, then [add the stack pack](/deploy-manage/deploy/cloud-enterprise/manage-elastic-stack-versions.md#ece-manage-elastic-stack-add) to your installation using the Cloud UI. 
+3. [Configure a snapshot repository](/deploy-manage/tools/snapshot-and-restore/cloud-enterprise.md) to enable snapshots and back up your data.
+ 
+ ## Perform the upgrade 
+
+ 1. [Log in to the Cloud UI](/deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md). 
+ 2. On the Deployments page, select your deployment.
+    
+    Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
+
+To upgrade a deployment:
+
+1. In the **Deployment version** section, click **Upgrade**.
+2. Select the version you're upgrading to. For example, `9.0.0`.
+3. Click **Upgrad**e, then **Confirm upgrade**. The new configuration takes a few minutes to create.
+
+    ::::{note} 
+    If any incompatibilities are detected when you attempt to upgrade, the UI provides a link to the Upgrade Assistant, which checks for deprecated settings in your cluster and indices and helps you resolve them. If there are any issues that would prevent a successful upgrade, the upgrade is blocked. After resolving the issues, return to the **Deployments** page and restart the upgrade.
+    ::::
+
+Security realm settings
+:   During the upgrade process, you are prompted to update the security realm settings if your user settings include a `xpack.security.authc.realms` value.
+
+    If the security realms are configured in `user_settings`, you’ll be prompted to modify the settings:
+
+    1. On the **Update security realm settings** window, edit the settings.
+    2. Click **Update settings**. If the security realm settings are located in `user_settings_override`, contact support to help you upgrade.
+
+## Next steps [next-steps-ece]
+
+Once you've successfully upgraded on {{ece}}, do the following: 
+
+* [Upgrade Elastic APM](../../../solutions/observability/apps/upgrade.md)
+* [Upgrade your ingest components](/deploy-manage/upgrade/ingest-components.md) 
```

Comment:
That one is clear, as the ingest components (beats, elastic agents, etc) are installed out of the ECE platform and are not managed by it. Maybe I'd add extra details about what we mean with ingest components, but up to you.

```suggestion
* [Upgrade your ingest components](/deploy-manage/upgrade/ingest-components.md), such as Elastic Agents, Logstash, or Beats.
```